modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -40,7 +50,7 @@ You require a Anypoint Service Mesh license to perform the installation. If you
 
 Before you can use Anypoint Service Mesh, you must have the following roles and permissions:
  
-* API Manager Environment Administrator permission or Organization Administrator role for your organization, to access the client ID and client secret credentials.
+* API Manager Environment Administrator or Organization Administrator permission for your organization, to access the client ID and client secret credentials.
```

Comment:
We need to revert this change. One thing is a permission, another one a role.

Reply:
Done! Thanks!